runtime/gc_glue_java/MetronomeDelegate.hpp

```diff
@@ -200,8 +200,9 @@ class MM_MetronomeDelegate : public MM_BaseNonVirtual
 	MMINLINE uintptr_t
 	scanPointerArraylet(MM_EnvironmentRealtime *env, fomrobject_t *arraylet)
 	{
+		UDATA const referenceSize = env->compressObjectReferences() ? sizeof(uint32_t) : sizeof(uintptr_t);
```

Comment:
this pattern repeats often. we should just have something like env->getObjectReferenceSize()